doc/source/serve/monitoring.md

```diff
@@ -456,6 +456,9 @@ The following metrics are exposed by Ray Serve:
      - * deployment
        * actor_id
      - The number of request scheduling tasks in the router that are undergoing backoff.
+   * - ``ray_serve_long_poll_host_bytes_sent`` [†]
```

Comment:
I'd suggest leaving this one undocumented given that it's very much in the weeds of implementation details

Reply:
Sure, I'm okay with that.